pkgs/applications/audio/rkrlv2/default.nix

```diff
@@ -1,26 +1,25 @@
-{ stdenv, fetchFromGitHub,
-automake, pkgconfig, lv2, fftw, cmake, xorg, libjack2, libsamplerate, libsndfile
-}:
+{ stdenv, fetchFromGitHub, pkg-config, lv2, fftw, cmake, libXpm
+, libXft, libjack2, libsamplerate, libsndfile }:
 
 stdenv.mkDerivation rec {
-  repo = "rkrlv2";
-  name = "${repo}-b2.0";
+  pname = "rkrlv2";
+  version = "beta_3";
 
   src = fetchFromGitHub {
     owner = "ssj71";
-    inherit repo;
-    rev = "beta_2";
-    sha256 = "128jcilbrd1l65c01w2bazsb21x78mng0jjkhi3x9crf1n9qbh2m";
+    repo = pname;
+    rev = version;
+    sha256 = "WjpPNUEYw4aGrh57J+7kkxKFXgCJWNaWAmueFbNUJJo=";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = with xorg; [ automake lv2 fftw cmake libXpm libjack2 libsamplerate libsndfile libXft ];
```

Comment:
Did you verify that automake is not needed? It makes no sense to use it with cmake but maybe the package does something really weird?

Reply:
Yes I built it and it is not needed.